### PR TITLE
Added support in WIX to install chef-client as a scheduled task.

### PR DESCRIPTION
**Pull request by Aliasgar16**
Issue ref: https://github.com/chef/chef/issues/4042

In-progress.

Signed-off-by: aliasgar16 <aliasgar.batterywala@msystechnologies.com>

### CODE REVIEW

File: omnibus/resources/chef/msi/source.wxs.erb [hunk 1]
```diff
@@ -173,12 +234,80 @@
...
+      <Publish Dialog="VerifyReadyDlg" Control="Back" Event="NewDialog" Value="WelcomeDlg" Order="3">Installed AND PATCH</Publish>
+
+      <Publish Dialog="MaintenanceWelcomeDlg" Control="Next" Event="NewDialog" Value="MaintenanceTypeDlg">1</Publish>
+
+      <Publish Dialog="MaintenanceTypeDlg" Control="ChangeButton" Event="NewDialog" Value="CustomizeDlg">1</Publish>
+      <Publish Dialog="MaintenanceTypeDlg" Control="RepairButton" Event="NewDialog" Value="VerifyReadyDlg">1</Publish>
+      <Publish Dialog="MaintenanceTypeDlg" Control="RemoveButton" Event="NewDialog" Value="VerifyReadyDlg">1</Publish>
+      <Publish Dialog="MaintenanceTypeDlg" Control="Back" Event="NewDialog" Value="MaintenanceWelcomeDlg">1</Publish>
+
+      <Dialog Id ="ChefServiceOptionSelectionDlg" Width ="270" Height ="130" Title ="Chef Service Options" NoMinimize ="no">
```
Comment (btm) [inline omnibus/resources/chef/msi/source.wxs.erb]: Let's add a text control here.

The installer can configure the Chef client to run periodically as either a scheduled task or a service. Using a scheduled task is recommended. For more information, see https://docs.chef.io/windows.html.

File: omnibus/resources/chef/msi/source.wxs.erb [hunk 2]
```diff
@@ -151,8 +196,24 @@
       <ComponentRef Id="ChefPSModulePath" />
     </Feature>
 
-    <Feature Id="ChefServiceFeature" Title="!(loc.FeatureServiceName)" Level="1000" AllowAdvertise="no">
-      <ComponentRef Id="ChefClientService" />
+    <Feature Id="ChefServiceOptions" Title="Chef Service Options" Level="1000" AllowAdvertise="no">
```
Comment (btm) [inline omnibus/resources/chef/msi/source.wxs.erb]: Let's change 'Chef Service Options' to 'Chef Unattended Execution Options'

File: omnibus/resources/chef/msi/source.wxs.erb (hunk 1 above)
Comment (btm) [inline omnibus/resources/chef/msi/source.wxs.erb]: Let's change 'Chef Service Options' to 'Chef Unattended Execution Options'

**Comment by Aliasgar16** [comment]
@btm I will make the required changes.

**Comment by Aliasgar16** [comment]
@btm I have made the required changes.

**Comment by Aliasgar16** [comment]
@btm @mwrock I supposed to test these new changes with newly built MSI (which @btm will supply me). So, I think this should not be merged until I test the changes.

**Pull request merged**